dashboard/app/models/pd/application/application_base.rb

```diff
@@ -230,11 +230,11 @@ def regional_partner_name
     end
 
     def school_name
-      user.try(:school_info).try(:effective_school_name).try(:titleize)
+      school.try(:name).try(:titleize)
     end
 
     def district_name
-      user.try(:school_info).try(:effective_school_district_name).try(:titleize)
+      District.find_by_id(school.try(:school_district_id)).try(:name).try(:titleize)
```

Comment:
We want `SchoolDistrict`, not `District` here (District is a leftover part of the old ops dashboard - deprecated and should be removed when we have a chance).

On that note, you can just use the [school_district](https://github.com/code-dot-org/code-dot-org/blob/3cb30d8eefe6c45bb6b3031a8b60da56dc6553f5/dashboard/app/models/school.rb#L33) association: 
```ruby
school.try(:school_district).try(:name).try(:titleize)
```